NET/ReadyToUseUI.iOS/Controller/FilterController.cs

```diff
@@ -8,24 +8,35 @@ namespace ReadyToUseUI.iOS.Controller
     public class FilterController : UIViewController
     {
         private FilterView ContentView;
-        private SBSDKImageFilterType Choice;
+        private Filter SelectedFilter = new Filter(SBSDKImageFilterType.None);
+        private SBSDKParametricFilter SelectedParametricFilter => GetParametricFilter(SelectedFilter);
         private SBSDKDocumentPage Temp;
 
-        private static readonly List<Filter> Filters = new List<Filter>
+        private readonly List<Filter> Filters = new List<Filter>
         {
-            new Filter("None", SBSDKImageFilterType.None),
-            new Filter("Low Light Binarization", SBSDKImageFilterType.LowLightBinarization),
-            new Filter("Low Light Binarization 2", SBSDKImageFilterType.LowLightBinarization2),
-            new Filter("Edge Highlight", SBSDKImageFilterType.EdgeHighlight),
-            new Filter("Deep Binarization", SBSDKImageFilterType.DeepBinarization),
-            new Filter("Otsu Binarization", SBSDKImageFilterType.OtsuBinarization),
-            new Filter("Clean Background", SBSDKImageFilterType.BackgroundClean),
-            new Filter("Color Document", SBSDKImageFilterType.ColorDocument),
-            new Filter("Color", SBSDKImageFilterType.Color),
-            new Filter("Grayscale", SBSDKImageFilterType.Gray),
-            new Filter("Binarized", SBSDKImageFilterType.Binarized),
-            new Filter("Pure Binarized", SBSDKImageFilterType.PureBinarized),
-            new Filter("Black & White", SBSDKImageFilterType.BlackAndWhite)
+            new Filter("New Filters", true),
+            new Filter(typeof(SBSDKScanbotBinarizationFilter)),
```

Comment:
@scan-mayank, the same comment I have made for android applies here too. We can simplify this example by including a delegate directly in the filter.